src/launcher.c

```diff
@@ -221,7 +221,7 @@ static void create_window()
                  SDL_WINDOWPOS_UNDEFINED,
                  0,
                  0,
-                 SDL_WINDOW_FULLSCREEN_DESKTOP | SDL_WINDOW_BORDERLESS
+                 SDL_WINDOW_FULLSCREEN_DESKTOP
```

Comment:
See #87 for an explanation of what I think is going on. `SDL_WINDOW_BORDERLESS` is redundant; `SDL_WINDOW_FULLSCREEN_DESKTOP` already means borderless.